backend/src/routes/api/profileRoutes.js

```diff
@@ -0,0 +1,23 @@
+import { StatusCodes } from 'http-status-codes';
+import Profile from '../../models/Profile.js';
+
+const userRoutes = async (router) => {
+  router.get('/users/:id', getUser);
+};
```

Comment:
there will be a conflict because this route will be used to retrieve User document

Please rename to ProfileRoutes